addons/api/addon/serializers/host-set.js

```diff
@@ -69,6 +69,15 @@ export default class HostSetSerializer extends ApplicationSerializer {
         delete json.attributes[key];
       }
     }
+
+    if (key === 'filter_string') {
+      const { filter_string } = snapshot.record;
+      if (filter_string) {
+        json.attributes.filter = json.attributes.filter_string;
+      }
+      delete json.attributes.filter_string;
```

Comment:
Why inspect the snapshot while subsequently operating on `json`?

Reply:
Ah I was just pattern matching what was previously done, is the implication I should just check the json directly and not bother interrogating the snapshot?

Reply:
If the value comes from `json.attributes`, I'd check that instead.

Reply:
Yup I updated it!